### ISSUE TITLE
Install Gymnasium (OpenAI Gym) on Windows | Resolve error "Failed building wheels for box2d-py"

### ISSUE THREAD

**Issue by engrsulaiman** [johnnycode8/gym_solutions#1]
@johnnycode   Hey, did you my below reply a day ago? I don't know why the reply is disappearing in the comment section. This my email please afitqadirectorate@gmail.com:
 Yes, https://networkattacksimulator.readthedocs.io is exactly what I was referring to. I tried following the documentation, but could not do it. Based on your Deep Q-Learning videos, I want to replicate this paper: https://www.mdpi.com/2076-3417/11/19/8823. So I will be very happy if you can help me. THANK YOU VERY MUCH IN ADVANCE

**Comment by engrsulaiman** [comment]
All my comments on your Youtube channel are disappearing, and I really needs your assistance on how replicate this paper: https://www.mdpi.com/2076-3417/11/19/8823. We can communicate through here please: afitqadirectorate@gmail.com. Waiting for your reply via my email. THANK YOU IN ADVANCE

**Comment by johnnycode8** [comment]
Not sure why your YouTube comments aren’t coming thru. How far did you get with nasim?

**Comment by engrsulaiman** [comment]
Thank you for your quick response. I have no idea on how to go about the
NASim. So I am a beginner, I wanted to know how install it, get acquainted
to different scenarios, then finally used your DQN tutorials to implement
the paper I sent the link to you. PLEASE HELP ME WITH MY RESEARCH.

**Comment by johnnycode8** [comment]
You should follow their tutorial on installation and how to get started. However, replicating the research paper will be very difficult. I don’t believe they supply their code used in the paper.

**Comment by engrsulaiman** [comment]
Yeah, I followed the tutorial, but could install it. I tried to get started
with scenarios, but I couldn't. As for the research paper, can't we think
of different scenarios, not necessarily their code? Thanks once again.
Regards.

**Comment by engrsulaiman** [comment]
A gentle eminder.
Thank you. Regards.

**Comment by johnnycode8** [comment]
I don't have a cyber background so I don't know of a different scenario.

**Comment by engrsulaiman** [comment]
Ok, thanks for the reply.
